ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/include.h

```diff
@@ -1,12 +1,12 @@
 #include "DIRECTX9Std.h"
-#include "Info/graphics_info.h" 
+#include "Info/graphics_info.h"
+#include "DX9shader.h"
 #include "../General/GSsprite.h"
 #include "../General/GSbackground.h"
-#include "DX9shader.h"
+#include "../General/GStiles.h"
```

Comment:
more ../

Reply:
Alright, I replaced `../General` with `Graphics_Systems/General` for all 4 of the `include.h` files in 8844907474400a89d66aeb6c43145f452151d01b

Reply:
Alright, I went a step further and removed `../General` from include statements occurring in sources that already in the folder `Graphics_Systems/General` as well in https://github.com/enigma-dev/enigma-dev/pull/1238/commits/ad7c513d30e4a4053cab11d8363c64f46466edcd